ci/premerge-build.sh

```diff
@@ -28,3 +28,6 @@ ${MVN} verify ${MVN_MIRROR} \
   -Dlibcudf.build.configure=true \
   -DUSE_GDS=ON -Dtest=*,!CuFileTest,!CudaFatalTest,!ColumnViewNonEmptyNullsTest \
   -DBUILD_TESTS=ON
+
+build_name=$($MVN help:evaluate -Dexpression=project.build.finalName -q -DforceStdout)
+. ci/check-cuda-dependencies.sh "target/${build_name}-cuda11.jar"
```

Comment:
Ideally we should not be hardcoding the cuda11 classifier here but instead querying for it, e.g.: querying pom for cuda.version or this will break when we drop CUDA 11 support eventually in the future.

Reply:
yes, I will update to include the cuda.version